app/questionnaire/questionnaire_schema.py

```diff
@@ -95,12 +95,10 @@ def get_default_answer(self, answer_id):
         except KeyError:
             question = block['question_variants'][0]['question']
 
-        answers = [
-            answer for answer in question['answers'] if answer['id'] == answer_id
-        ]
+        answer = question['answers'][0]
```

Comment:
This will always choose the first answer. It should loop through all answers looking for one that matches the `answer_id` in the method call.